modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -20,7 +20,13 @@ import ClayLayout from '@clayui/layout';
 import ClayModal from '@clayui/modal';
 import {FrontendDataSet} from '@liferay/frontend-data-set-web';
 import classNames from 'classnames';
-import {fetch, navigate, openModal} from 'frontend-js-web';
+import {
+	fetch,
+	navigate,
+	openConfirmModal,
+	openModal,
+	openToast,
```

Comment:
Uh, I spammed several `Liferay.Util.openToast`s in the code :grimacing:  

This is the way :+1: 

Feel free to change existing usages when convenient. Not a blocker for this PR.